virttest/ppm_utils.py

```diff
@@ -26,6 +26,12 @@
                     'and Windows guest BSOD detection are disabled. '
                     'In order to enable it, please install python-imaging or '
                     'the equivalent for your distro.')
+# Prevent logs pullotion
+if Image:
```

Comment:
Not a big deal, but PEP8 says that "Comparisons to singletons like None should always be done with is or is not, never the equality operators", so this could be `if Image is not None`.

Reply:
Thanks, will update it.